src/compiler/types.ts

```diff
@@ -10410,3 +10524,11 @@ export interface SyntacticTypeNodeBuilderResolver {
     requiresAddingImplicitUndefined(parameter: ParameterDeclaration | JSDocParameterTag, enclosingDeclaration: Node | undefined): boolean;
     isDefinitelyReferenceToGlobalSymbolObject(node: Node): boolean;
 }
+/** @internal */
+

```

Comment:
Stray empty line here; I've seen a couple of these in the diff.

Reply:
These and the incorrect imports are moveToFile bugs. I noticed some as I was making the edits, but I didnt manage to fix all of the extra spaces and imports :,(